js/src/panes/pane.map.js

```diff
@@ -256,18 +256,19 @@ Wu.MapPane = Wu.Pane.extend({
     },
 
     _onMove : function () {
```

Comment:
https://github.com/mapic/mapic.js/blob/master/js/src/models/model.project.js#L357
This previous event was actually calling API for `setSlug` . But as we know now , this event was not get called . So I think we need to think and refactor this Update Project Flow. Like - separating - Update Project Name and details and Slug. I will think and come up with a refactor